site2/docs/io-elasticsearch-sink.md

```diff
@@ -16,11 +48,45 @@ The configuration of the ElasticSearch sink connector has the following properti
 |------|----------|----------|---------|-------------|
 | `elasticSearchUrl` | String| true |" " (empty string)| The URL of elastic search cluster to which the connector connects. |
 | `indexName` | String| true |" " (empty string)| The index name to which the connector writes messages. |
+| `schemaEnable` | Boolean | false | false | Turn on the Schema Aware mode. |
+| `createIndexIfNeeded` | Boolean | false | false | Manage index if missing. |
+| `maxRetries` | Integer | false | 1 | The maximum number of retries for elasticsearch requests. Use -1 to disable it.  |
+| `retryBackoffInMs` | Integer | false | 100 | The base time in milliseconds to wait when retrying an elasticsearch request. |
+| `maxRetryTimeInSec` | Integer| false | 86400 | The maximum retry time interval in seconds for retrying an elasticsearch request.. |
+| `bulkEnabled` | Boolean | false | false | Enable the elasticsearch bulk processor to flush write requests based on the number or size of requests, or after a given period. |
+| `bulkActions` | Integer | false | 1000 | The maximum number of actions per elasticsearch bulk request. Use -1 to disable it. |
+| `bulkSizeInMb` | Integer | false |5 | The maximum size in megabytes of elasticsearch bulk requests.Use -1 to disable it. |
+| `bulkConcurrentRequests` | Integer | false | 0 | The maximum number of in flight elasticsearch bulk requests. The default 0 allows the execution of a single request. A value of 1 means 1 concurrent request is allowed to be executed while accumulating new bulk requests. |
+| `bulkFlushIntervalInMs` | Integer | false | -1 | The bulk flush interval flushing any bulk request pending if the interval passes. Default is -1 meaning not set. |
+| `compressionEnabled` | Boolean | false |false | Enable elasticsearch request compression. |
+| `connectTimeoutInMs` | Integer | false |5000 | The elasticsearch client connection timeout in milliseconds. |
+| `connectionRequestTimeoutInMs` | Integer | false |1000 | The time in milliseconds for getting a connection from the elasticsearch connection pool. |
+| `connectionIdleTimeoutInMs` | Integer | false |5 | Idle connection timeout to prevent a read timeout. |
+| `keyIgnore` | Boolean | false |true | Whether to ignore the record key to build the Elasticsearch document _id. If primaryFields is defined, the connector extract the primary fields from the payload to build the document _id. If no primaryFields are provided, elasticsearch auto generates a random document _id. |
```

Comment:
add backticks for _id? (for example,`_id`?)

Reply:
sure. good idea